pandas/tests/io/parser/test_common.py

```diff
@@ -1814,13 +1814,16 @@ class InvalidBuffer(object):
         parser.read_csv(InvalidBuffer())
 
 
-def test_invalid_file_buffer_mock(all_parsers, mock):
+def test_invalid_file_buffer_mock(all_parsers):
     # see gh-15337
     parser = all_parsers
     msg = "Invalid file path or buffer object type"
 
+    class Mock():
```

Comment:
Personally would rather just name this `Foo` so as not to cause confusion with the real `Mock` class